### PR TITLE
Update release instructions

### PR DESCRIPTION
**Pull request by idodod**

### CODE REVIEW

File: release/README.md [hunk 1]
```diff
@@ -18,19 +18,24 @@
...
   | MacOS (M1)    | [![Build status](https://badge.buildkite.com/10a7331b2032fcc9f7f311c5218d12c1a18c317cd7fc9270ba.svg)](https://buildkite.com/earthly-technologies/earthly-m1-scheduled)
   | Windows (WSL) | [![Build status](https://badge.buildkite.com/19d9cf7fcfb3e0ee45adcb29abb5ef3cfcd994fba2d6dc148c.svg)](https://buildkite.com/earthly-technologies/earthly-windows-scheduled)
-* Run
-  ```bash
-  env -i HOME="$HOME" PATH="$PATH" SSH_AUTH_SOCK="$SSH_AUTH_SOCK" RELEASE_TAG="$RELEASE_TAG" USER="$USER" PRERELEASE="$PRERELEASE" ./release.sh
-  ```
-* Merge branch `main` into `docs-0.7`
+  * Run
+    ```bash
+    cd release
```
Comment (alexcb) [inline release/README.md:31]: I don't think this is needed -- the README is already in the `release` directory.

File: release/README.md [hunk 2]
```diff
@@ -18,19 +18,24 @@
...
+    env -i HOME="$HOME" PATH="$PATH" SSH_AUTH_SOCK="$SSH_AUTH_SOCK" RELEASE_TAG="$RELEASE_TAG" USER="$USER" PRERELEASE="$PRERELEASE" ./release.sh
+    ```
+  * Merge branch `main` into `docs-0.7` (via `git`):
+    ```shell
+      git checkout docs-0.7 && git merge main && git push
```
Comment (alexcb) [inline release/README.md:36]:
```suggestion
      git checkout docs-0.7 && git pull && git merge main && git push
```

File: release/README.md [hunk 3]
```diff
@@ -59,8 +64,17 @@
   * [buildkit-standalone.md](../docs/docker-images/buildkit-standalone.md)
 * Commit updated version changes to `docs-0.7`.
 * Merge `docs-0.7` into `main`.
+  ```shell
+    git checkout main && git merge docs-0.7 && git push
+    ```
+  * Note: If you don't have permissions to push directly to `main` branch, do the following:
+    * `git checkout -b soon-to-be-main && git push origin soon-to-be-main`
+    * Open a PR against the new branch and get it approved.
```
Comment (alexcb) [inline release/README.md:72]:
```suggestion
    * Open a PR against the new branch and get it approved; IMPORTANT: don't squash-merge via github
```

**Review by alexcb — APPROVED** [review]
thanks for updating this; just a few minor nits.

File: release/README.md (hunk 1 above)
Comment (idodod) [reply release/README.md:31]: The readme is, but for example when I was using the readme I opened it in my browser and my terminal was pointing to the repo root. also other commands such as `./earthly` assume your terminal is in root.

Comment (alexcb) [reply release/README.md:31]: ah that's a good point regarding `./earthly`, feel free to ignore this comment then.

**Pull request merged**